dbgpt/storage/knowledge_graph/knowledge_graph.py

```diff
@@ -172,3 +173,7 @@ def delete_vector_name(self, index_name: str):
 
         logger.info("Drop triplet extractor")
         self._triplet_extractor.drop()
+
+    def delete_by_ids(self, ids: str) -> List[str]:
+        self._graph_store.delete_document(chunk_ids = ids)
```

Comment:
The `delete_document` should be implemented in `CommunityStoreAdapter` instead of directly modifying the API of `GraphStore`.